app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -9,7 +9,8 @@
 
 - when 'UsersRole'
   - users_role = current_or_last_object_state(version.item_type, version.item_id)
-  - role = Role.find_by(id: users_role.role_id) if users_role
+  - if users_role
+    - role = Role.find_by(id: users_role.role_id) || PaperTrail::Version.where(item_type: 'Role', item_id: version.item_id).where(id: users_role.role_id)
```

Comment:
could this still be `nil`? should we maybe add a `.try` in `role.name`? 🤔

Reply:
yes right using .try will be better.